gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -34,67 +34,6 @@
 @AutoValue
 public abstract class GoogleCloudStorageOptions {
 
-  /** Default setting for enabling use of GCS gRPC API. */
-  public static final boolean DEFAULT_GRPC_ENABLED = false;
-
-  /** Default setting for enabling use of the Traffic Director for GCS gRPC API. */
-  public static final boolean DEFAULT_TRAFFIC_DIRECTOR_ENABLED = true;
-
-  /** Default setting to prefer DirectPath for gRPC. */
-  public static final boolean DEFAULT_DIRECT_PATH_PREFERRED = true;
-
-  /** Default root URL for Cloud Storage API endpoint. */
-  public static final String STORAGE_ROOT_URL_DEFAULT = Storage.DEFAULT_ROOT_URL;
-
-  // Default root URL for Cloud Storage gRPC API endpoint. */
-  public static final String DEFAULT_GCS_GRPC_SERVER_ADDRESS =
-      StorageProto.getDescriptor()
-          .findServiceByName("Storage")
-          .getOptions()
-          .getExtension(ClientProto.defaultHost);
-
-  /** Default service Path for Cloud Storage API endpoint. */
-  public static final String STORAGE_SERVICE_PATH_DEFAULT = Storage.DEFAULT_SERVICE_PATH;
-
-  /** Default setting for enabling auto-repair of implicit directories. */
-  public static final boolean AUTO_REPAIR_IMPLICIT_DIRECTORIES_DEFAULT = true;
-
-  /**
-   * Default setting for the length of time to wait for empty objects to appear if we believe we are
-   * in a race with multiple workers.
-   */
-  public static final int MAX_WAIT_MILLIS_FOR_EMPTY_OBJECT_CREATION = 3_000;
```

Comment:
why changing this? I think there the existing structure is more readble.

Reply:
They provide no value as we already have default instance of option class itself, clutter the code, and they are not consistent between option classes.